src/components/organize/TagsManager/TagDialog/ColorPicker.tsx

```diff
@@ -0,0 +1,79 @@
+import ReplayIcon from '@material-ui/icons/Replay';
+import { useIntl } from 'react-intl';
+import { Box, InputAdornment, TextField } from '@material-ui/core';
+
+import { DEFAULT_TAG_COLOR } from '../utils';
+
+const randomColor = () => {
+  return Math.floor(Math.random() * 16777215).toString(16);
+};
+
+const hexRegex = new RegExp(/(^[0-9a-f]{6}$)|(^[0-9a-f]{3}$)/i);
```

Comment:
I just realized the API does not accept 3-digit colors, so this needs to change to only validate six-digits (or expand the three-digit ones before submitting them to the API).

Reply:
Will do